server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/EndpointSubsystemAdd.java

```diff
@@ -41,14 +37,20 @@
 import org.jboss.msc.service.ServiceBuilder;
 import org.jboss.msc.service.ServiceName;
 
+import static org.infinispan.server.endpoint.subsystem.ModelKeys.HOTROD_CONNECTOR;
+import static org.infinispan.server.endpoint.subsystem.ModelKeys.MEMCACHED_CONNECTOR;
+import static org.infinispan.server.endpoint.subsystem.ModelKeys.REST_CONNECTOR;
+import static org.infinispan.server.endpoint.subsystem.ModelKeys.ROUTER_CONNECTOR;
```

Comment:
@tristantarrant Do we have a consensus on where these static imports should go?  I know that eclipse by default likes the top and intellij like the bottom (or something like that).  Just would be nice if we could make sure we do the same spot each time.  Maybe we should update the various code styles to be the same?